charts/ocis/values.yaml

```diff
@@ -557,6 +561,18 @@ services:
   # -- SEARCH service.
   # @default -- see detailed service configuration options below
   search:
+    # -- Search Extractor settings.
+    # @default -- see detailed search extractor configuration options below
+    extractor:
+      # -- Configures the search extractor type to be used. Possible extractors:
+      # - `basic`: the default search extractor.
+      # - `tika`: the Tika search extractor. If set to this value, additional settings in the `tika` section apply.
+      type: basic
+      tika:
+        # -- Set the URL to Tika. Only applicable if `services.search.extractor.type` == `tika`.
```

Comment:
Do we expect more tika setting in the future? If not we might skip the type and just switch to tika if a tika url is set.

Reply:
Ok in this case it makes sense.